doc/README.md

```diff
@@ -34,8 +34,7 @@ Drag DigiByte Core to your applications folder, and then run DigiByte Core.
 
 * See the documentation at the [DigiByte Wiki](https://en.digibyte.it/wiki/Main_Page)
 for help and more information.
-* Ask for help on [#digibyte](http://webchat.freenode.net?channels=digibyte) on Freenode. If you don't have an IRC client use [webchat here](http://webchat.freenode.net?channels=digibyte).
-* Ask for help on the [DigiByteTalk](https://digibytetalk.org/) forums, in the [Technical Support board](https://digibytetalk.org/index.php?board=4.0).
+* Ask for help on [Gitter](https://gitter.im/DigiByte-Core/protocol). If you don't have a Gitter client [find one here](https://gitter.im/apps).
```

Comment:
I agree with @mctrivia that this would be better off being dgbsupport, because we've seen people told again and again not to go to Gitter for support / general chat / price discussion, no sense to be leading them there now.. On line #70 where we reference development then it makes sense to have it there for sure. Think it should be ACK'd after.

Reply:
Good evening and thanks for the feedback @mctrivia and @ChillingSilence ,

This PR is for support related specifically to developing against the core protocol code versus more generalized support which you are referring to.   While the readme does contain a section related to running the client, the majority of it's contents are written for a developer audience.

We absolutely want to encourage developers to converse on Gitter regarding getting a local development environment set up, which is what this documentation is for. 

Having said that, I am not opposed to @mctrivia opening a separate pull request to add a reference to DGBSupport .  This PR's focus was to replace references to IRC rather than address broader support channel needs.  I think that would be absolutely appropriate for it's own PR.

Thanks for the feedback!

Reply:
This is under "Running" though, it's not where we want people to be asking for general support in Gitter. The other references seem fine, but look back at the header that this is under, and we're offering support for 'running' the node, which is unrelated to development. In order to keep it separated, I still don't think this here should mention Gitter. Support for "Why is it crashing when I have 2GB of RAM" is unrelated to development, but *is* support-related.